packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -207,6 +207,8 @@ enum _PickerColumnType {
   minute,
   // AM/PM column in time and dateAndTime mode.
   dayPeriod,
+  // Two points column in time and dateAndTime mode.
+  twoPoints,
```

Comment:
Nit: can we rename this to timeSeperator? It's a little more consistent with how we refer to it elsewhere.

Reply:
Okay I'll push new updates then